packages/tradershub/tailwind.config.js

```diff
@@ -72,6 +72,30 @@ module.exports = {
             fontFamily: {
                 sans: ['IBM Plex Sans', 'sans-serif'],
             },
+            height: {
+                '10v': '10vh',
+                '20v': '20vh',
+                '30v': '30vh',
+                '40v': '40vh',
+                '50v': '50vh',
+                '60v': '60vh',
+                '70v': '70vh',
+                '80v': '80vh',
+                '90v': '90vh',
+                '100v': '100vh',
+            },
+            width: {
+                '10v': '10vw',
+                '20v': '20vw',
+                '30v': '30vw',
+                '40v': '40vw',
+                '50v': '50vw',
+                '60v': '60vw',
+                '70v': '70vw',
+                '80v': '80vw',
+                '90v': '90vw',
+                '100v': '100vw',
+            },
```

Comment:
Do we need to use values like `20vw` or `10vh` in our styles? Tailwind CSS provides built-in styles for similar properties. Alternatively, we can temporarily hard code values until Tradershub design adopts semantic values.

Reply:
As i checked we dont have vh and vw in built in styles. Thats why i added

Reply:
Will remove it for now till we have better clarification